### PR TITLE
Time utils

### LINKED ISSUE #9: Better way to account for timezone difference

### PR DESCRIPTION
**Pull request by not7cd**
Resolve #9 
client.CreateEvent() should get time in POSIX in ms.
Add logging utility.
[Waiting for answer](https://twitter.com/not7CD/status/904417306520014849)

### CODE REVIEW

File: utility.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import yaml
 
 def readYAML(filename):
+    """Returns dict of parsed data from file"""
     with open(filename, 'r') as stream:
         try:
-            parsedData = (yaml.load(stream))
-            return parsedData
+            parsed_data = (yaml.load(stream))
+            return parsed_data
```
Comment (allgreed) [inline utility.py]: We can stick to PEP8, good idea ;)

File: docs/sample_event_file.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Every line must start with hash
 
 # CAUTION: Be sure to wrap name and description in quotes
---- 
+---
```
Comment (allgreed) [inline docs/sample_event_file.yml]: Szanuję ^^

File: timeutils.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    # TODO: Test meetup API for time
+    # make original datetime
+    # meetup_tz = timezone.utc
+    # meetup_dt = dt.astimezone(tz=MEETUP_TZ)
+    # meetup_dt = dt.replace(tzinfo=MEETUP_TZ)
+    # logger.debug(meetup_dt)
+
+    # logger.debug((meetup_dt.astimezone(tz=timezone.utc)))
+
+    logger.debug("%s, %s" % (dt, dt.timestamp()))
```
Comment (allgreed) [inline timeutils.py]: Plox use format...

File: timeutils.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import logging
+from datetime import timedelta, timezone, datetime
+logger = logging.getLogger(__name__)
+
+# Nowy York xD
+MEETUP_TZ = timezone(-timedelta(hours=4))
```
Comment (allgreed) [inline timeutils.py]: ???
That's confusing...

File: docs/sample_event_file.yml (hunk 2 above)
Comment (not7cd) [reply docs/sample_event_file.yml]: Automated beautification

File: timeutils.py (hunk 4 above)
Comment (not7cd) [reply timeutils.py]: Reference const, Meetup is NYC based company. It looks like timezone shouldn't matter after all.

File: timeutils.py (hunk 3 above)
Comment (not7cd) [reply timeutils.py]: This outputs
```
2017-09-03 21:16:28,310 - timeutils - DEBUG - 2017-08-22 18:00:00, 1503417600.0
2017-09-03 21:16:28,310 - timeutils - INFO - change timezone
2017-09-03 21:16:28,310 - timeutils - DEBUG - 2017-08-22 18:00:00+02:00, 1503417600.0
```

Comment (allgreed) [reply timeutils.py]: So?

File: timeutils.py (hunk 4 above)
Comment (allgreed) [reply timeutils.py]: @not7cd I mean -timedelta

Comment (not7cd) [reply timeutils.py]: https://docs.python.org/3/library/datetime.html#datetime.timezone

**Comment by allgreed** [comment]
Ok, it works. Comencing merge :D

**Pull request merged**